io.catenax.quality_task_attachment/2.0.0/QualityTaskAttachment.ttl

```diff
@@ -0,0 +1,172 @@
+
+:ListOfFiles a samm-c:List ;
+   samm:preferredName "List of files"@en ;
+   samm:description "List of files attached to a quality task"@en ;
+   samm:dataType :File .
+
+:File a samm:Entity ;
+   samm:preferredName "File"@en ;
+   samm:description "One file attached to the quality task"@en ;
+   samm:properties ( :fileName [ samm:property :schema; samm:optional true ] :filePath :sizeInKb :fileDescription :fileExtension ) .
+
+:fileName a samm:Property ;
+   samm:preferredName "file name"@en ;
+   samm:description "The name of the described file"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Histogramm_data.csv" .
+
+:schema a samm:Property ;
+   samm:preferredName "schema"@en ;
+   samm:description "schema definition for file"@en ;
+   samm:characteristic :Schema .
+
+:filePath a samm:Property ;
+   samm:preferredName "File Path"@en ;
+   samm:description "Indicates where the file is located within the ZIP-folder"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "/subfolder/Histogramm_data.csv" .
+
+:sizeInKb a samm:Property ;
+   samm:preferredName "Size in Kilobyte"@en ;
+   samm:description "Indicates the size of the file"@en ;
+   samm:characteristic :SizeInKb ;
+   samm:exampleValue "615"^^xsd:positiveInteger .
+
+:fileDescription a samm:Property ;
+   samm:preferredName "File Description"@en ;
+   samm:description "A description of the file content"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Price list for various components" .
+
+:fileExtension a samm:Property ;
+   samm:preferredName "File Extension"@en ;
+   samm:description "Indicates which file format the file has"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "csv" .
+
+:Schema a samm:Characteristic ;
+   samm:preferredName "Schema"@en ;
+   samm:description "Describes the schema of the values contained in the file"@en ;
+   samm:dataType :SchemaDefinition .
+
+:SizeInKb a samm-c:Measurement ;
+   samm:preferredName "Size in Kilobyte"@en ;
+   samm:description "Indicates the size of the file"@en ;
+   samm:dataType xsd:positiveInteger ;
+   samm-c:unit unit:kilobyte .
+
+:SchemaDefinition a samm:Entity ;
+   samm:preferredName "Schema"@en ;
+   samm:description "Schema for the file entities"@en ;
+   samm:properties ( [ samm:property :decimalSeperator; samm:optional true ] [ samm:property :delimiter; samm:optional true ] :variables ) .
+
+:decimalSeperator a samm:Property ;
+   samm:preferredName "Decimal Separator Enumeration"@en ;
+   samm:description "Enumeration types to be used for the decimal separators"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "\"comma\", \"dot\"" .
+
+:delimiter a samm:Property ;
+   samm:preferredName "Delimiter"@en ;
+   samm:description "Indicates whether a semicolon, comma or tab is used as a delimiter between data points"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "semicolon" .
+
+:variables a samm:Property ;
```

Comment:
Property and the referenced Characteristic should not have the same name

Reply:
Changed naming